db/migrate/20160202205449_add_organization_to_user.rb

```diff
@@ -1,8 +1,9 @@
 class AddOrganizationToUser < ActiveRecord::Migration
   def up
     add_reference :users, :organization
-    User.where(organization_id: nil).update_all(organization_id: Organization.first.id)
-    Player.where(organization_id: nil).update_all(organization_id: Organization.first.id)
+    # NOTE: Should we extract these two lines below into a rake task?
```

Comment:
Sorry for the delay in replying! Please, extract these lines to a rake task, they shouldn't be here. :/